sentry-android-distribution/src/main/java/io/sentry/android/distribution/internal/DistributionIntegration.kt

```diff
@@ -0,0 +1,18 @@
+package io.sentry.android.distribution.internal
```

Comment:
I'm thinking if we should move this out of the `internal` package? Because this will be the entry point for this submodule/integration and be accessed publicly from within `sentry-andorid-core`